### ISSUE TITLE
[Visual requirement - Make code Arcade- Projects-Language]: Luminosity ratio of the focus indicator at 'Translate the editor' doesn't meet the minimum contrast ratio 3:1.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6116]
### User Experience:
Keyboard dependent and low vision people will get impacted as they won't be able to perceive the focus indicator clearly if luminosity ratio of the focus indicator does not meet minimum contrast ratio. The end user cannot select/unselect the utilization checkbox and cannot navigate the page efficiently.

**Note:** User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/#
2. Navigate to 'Settings' icon and navigate to 'Language' and select it.
3. Navigate to 'Translate the editor'. 
4. Verify the luminosity ratio of the focus indicator at 'Translate the editor'. 

### Actual Result:
**Issue:** Luminosity ratio of the focus indicator at 'Translate the editor' doesn't meet the minimum contrast ratio 3:1.

**Observation:** Actual luminosity ratio of the checkboxes is 1.6:1

### Expected Result:
Luminosity ratio of 'Translate the editor' should meet minimum contrast ratio 3:1.

### MAS Reference:
[MAS 1.4.11 – Non-text Contrast](https://aka.ms/MAS1.4.11)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**

NA

**UIA/DOM details available:** 

NA
### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
**OS Version:** Windows 11 24H2 (OS Build:26100.3037)
**Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
**Test Matrix:** Chrome + NVDA
**URL:** https://arcade.makecode.com/projects

### Attachment:

<img width="1362" alt="Image" src="https://github.com/user-attachments/assets/abb83a92-81f5-4d9c-a406-de339dff359a" />

**Comment by srietkerk** [comment]
This has been fixed with the new theme changes. No further work needed other than a beta bump/release.
